airflow/configuration.py

```diff
@@ -324,17 +344,47 @@ def run_command(command):
 # how often the scheduler should run (in seconds).
 scheduler_heartbeat_sec = 5
 
+# after how much time should the scheduler terminate in seconds
+# -1 indicates to run continuously (see also num_runs)
+run_duration = -1
+
+# after how much time a new DAGs should be picked up from the filesystem
+min_file_process_interval = 0
+
+dag_dir_list_interval = 300
```

Comment:
this is the amount of time the scheduler should wait before refreshing the dag list (by looking into the dag directory)